spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/pom.xml

```diff
@@ -14,21 +14,21 @@
     </parent>
 
     <dependencies>
-        <dependency>
+<!--        <dependency>
```

Comment:
Huh?

Reply:
Snyk troubleshooting on the wrong branch, sorry.